build/_k-standard-goals.shade

```diff
@@ -30,6 +30,8 @@ default Quiet='${ false }'
     Configuration = "Debug";
     E("Configuration", Configuration);
   }
+  
+  Console.WriteLine("Build v2: " + IsBuildV2);
```

Comment:
I'll keep it. It makes easy to debug logs since I will enable it for some repos only. But I'll change it to `Log.Info`